players/finding-mods.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Finding Trustworthy Mods
+description: A guide on how to find Fabric mods using trustworthy sources.
+---
+
+# Finding Trustworthy Mods
+
+Firstly, trust is subjective, and you should always use your own judgement when downloading mods. However, there are some things you can do to help you find trustworthy mods.
+
+## 1. Use a source that is known to be trustworthy.
+
+The majority of authors publish their mods to [Modrinth](https://modrinth.com/mods?g=categories:%27fabric%27) and [CurseForge](https://www.curseforge.com/minecraft/search?page=1&pageSize=20&sortType=1&class=mc-mods&gameFlavorsIds=4), however some may choose to upload it on their personal websites, or on other platforms, such as a GitHub repository.
+
+These platforms have moderation teams in place that check that mods are what they say they are, and that they do not contain any malicious code. They also have a reporting system in place, so that if you do find a malicious mod, you can report it to the moderation team, and they will take action relatively quickly.
+
+## 2. Check with others!
+
+If you are downloading a mod from a source that is not known to be trustworthy, you should check with others to see if they have downloaded the mod before from the location you're downloading from, and if they have had any issues with it.
+
+If in doubt, you are welcome to ask in the [Fabric Discord](https://discord.gg/v6v4pMv) in the `#player-support` channel.
+
+## 3. Avoid common malware sites!
+
+There are a lot of sites out there that claim to have mods for Minecraft, but are actually just malware sites. You should avoid these sites at all costs.
+
+Some examples of sites that are known to contain malware are:
+
+- `9minecraft.net`
+- `10minecraft.ru`
+- `creeper.gg`
+
+A full comprehensive list of these sites can be found [here, courtesy of Stop Mod Reposts](https://stopmodreposts.org/).
+
```

Comment:
Remove this, this will only boost their SEO rankings IMO. And possibly give a false sense of security for the ones not listed.

SMR is no longer maintained, again a false sense of security.